go-controller/pkg/cni/OCP_HACKS.go

```
@@ -73,29 +76,45 @@ func setupIPTablesBlocks(netns ns.NetNS, ifInfo *PodInterfaceInfo) error {
 
 // OCP HACK: wait for OVN to fully process the new pod
 func ofctlExec(args ...string) (string, error) {
-	args = append([]string{"--timeout=30"}, args...)
-	output, err := runner.Command("ovs-ofctl", args...).CombinedOutput()
+	args = append([]string{"--timeout=30", "--no-stats", "--strict"}, args...)
+	var stdout, stderr bytes.Buffer
+	cmd := runner.Command("ovs-ofctl", args...)
+	cmd.SetStdout(&stdout)
+	cmd.SetStderr(&stderr)
```

you don't need all this; just use `Output` instead of `CombinedOutput` above and then you'll only get stdout rather than stdout+stderr

---

@danwinship we are capturing the stderr so that we can report the error in case of failure, whether it's timeout or error in query. And the reason i separated them out is to avoid a grep (String operation which is not needed) on CombinedOut.